website/for_teachers.py

```diff
@@ -1140,9 +1199,13 @@ def create_adventure(self, user):
             "date": utils.timems(),
             "creator": user["username"],
             "name": name,
+            "classes": class_id,
```

Comment:
the classes field you use should be a list of IDs, you change it later if i wanna add the current adventure to other classes, but since it indicates a list, make sure it is initially.